autotest.pm

```diff
@@ -52,6 +51,7 @@ loadtest is called.
 
 =cut
 
+no warnings 'redefine';    # unclear why this is needed
```

Comment:
I think that's the `use bmwqemu 'diag';` in `Utils.pm`. There's one `diag` call that isn't namespaced.

Reply:
See #1549 for a fix